Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/BankSelectorTokenizationDelegate.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  BankSelectorTokenizationModel.swift
+//  PrimerSDK
+//
+//  Created by Alexandra Lovin on 16.11.2023.
+//
+
+import Foundation
+
```

Comment:
I'm not sure on the use of `Delegate` in the naming here. Delegates are usually an object we want to inform that something has happened, which I don't think quite fits here.

What would you think about:

`TokenizationProviding`
`BankSelectorTokenizationProviding: TokenizationProviding`
`WebRedirectTokenizationProviding: TokenizationProviding`

In my mind the name for the `BankSelectorTokenizationViewModel` would be `BankSelectionTokenizationProvider` - but that would be a breaking change, so we cant go that far right now.

Reply:
Looking deeper, `WebRedirectTokenizationProviding` - does not handle any tokenization, so this could indeed be a Delegate. The only outlier is `handlePaymentMethodTokenData` - could it be in the other one?

Reply:
Sure, we can use the providing as part of the protocol. Will look into how I can move the handlePaymentMethodTokenData to the bankselector .

Reply:
See with latest changes